src/commands/workspace/promote.ts

```diff
@@ -18,8 +18,8 @@ const conflictFlagDescription = [
 
 const conflictResolutionFlag = flags.string({
   description: conflictFlagDescription,
-  options: ['master', 'mine', 'abort'],
-  default: 'master',
+  options: ['masterWins', 'mineWins', 'abort'],
```

Comment:
What about keeping the flag names as they are and map them to the API values afterwards?